loopchain/channel/channel_inner_service.py

```diff
@@ -627,6 +628,24 @@ async def block_height_response(self, peer: str, height: int):
         """
         self._channel_service.consensus_runner.update_status(peer, height)
 
+    @message_queue_task(priority=255)
+    async def block_request(self, peer: str, height: int):
+        """Handle BlockRequest.
+
+        Triggers to reply target block by AnnounceUnconfirmedBlock
+
+        Note that BlockRequest allows only Block 1.0+
+        """
+        blockchain = self._channel_service.block_manager.blockchain
+        block: Block_V1_0 = blockchain.find_block_by_height(height)
+
+        block_dumped = self._channel_service.block_manager.blockchain.block_dumps(block)
+        block_send = loopchain_pb2.BlockSend(block=block_dumped, round_=block.header.round, channel=ChannelProperty().name)
```

Comment:
@noveljava 
I'm wondering that round value is needed or not during Consensus Sync.